src/Header/CookieBanner/CookieBanner.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import Cookies from "js-cookie";
+
+import styles from "./CookieBanner.module.scss";
+
+export const CookieName = "seen_cookie_message";
+
+// Increment this and release a new version to force users to re-accept the message
+export const CookieMessageVersion = 2;
+
+export default class CookieBanner extends Component {
+	constructor(props) {
+		super(props);
+
+		const cookieValue = Cookies.getJSON(CookieName);
+
+		// "yes" was used for the previous versoin of TopHat
```

Comment:
Typo in comments - `versoin` - sorry pedantic!

Reply:
Good point, will fix!